ansible/roles/preflight-invalidation/defaults/main.yml

```diff
@@ -0,0 +1,7 @@
+---
+preflight_invalidation_namespace: preflight-invalidation
+
+preflight_invalidation_private_key_local_path: ../../vaults/{{ env }}/operator-pipeline.key
+preflight_invalidation_private_cert_local_path: ../../vaults/{{ env }}/operator-pipeline.pem
+
+preflight_invalidation_image_pull_spec: quay.io/redhat-isv/operator-pipelines-images:latest
```

Comment:
We probably want to use the `released` tag here rather than `latest`, at least for prod.
For development, we can always override it at run time when running the playbook.